Generics.md

```diff
@@ -73,21 +73,21 @@ Commit and distribute to CMR as done in the addition steps.
 ## Generic Document Pipeline API Endpoints
 
 The API endpoints are generated from a pre-configured list of document types. The type is singular in
-ingest and plural in search. The examples below use the "orderoption"/"orderoptions" type, with a
-native ID of "orderoption-1".
+ingest and plural in search. The examples below use the "order-option"/"order-options" type, with a
+native ID of "order-option-1".
 
 #### ingest
-	curl -v -XPOST -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/orderoption/orderoption-1?provider=PROV1" -d @orderoption-1.json
-	curl -v -XPUT -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/orderoption/orderoption-1?provider=PROV1" -d @orderoption-1.json
-	curl -v -XGET -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/orderoption/orderoption-1?provider=PROV1"
-	curl -v -XDELETE -H "$TOKEN" "https://cmr.earthdata.nasa.gov/ingest/orderoption/dataqualitysummary-1?provider=PROV1"
+	curl -v -XPOST -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/order-option/order-option-1?provider=PROV1" -d @order-option-1.json
+	curl -v -XPUT -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/order-option/order-option-1?provider=PROV1" -d @order-option-1.json
+	curl -v -XGET -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/order-option/order-option-1?provider=PROV1"
+	curl -v -XDELETE -H "$TOKEN" "https://cmr.earthdata.nasa.gov/ingest/order-option/order-option-1?provider=PROV1"
```

Comment:
Shouldn't these interface be changed now? I know the following works: 
(curl+ -v -XPOST -H "Echo-Token: mock-echo-system-token"
         -H "Content-Type:application/vnd.nasa.cmr.umm+json"
         "http://localhost:3002/providers/PROV1/serviceentries/serviceentry-8671"
         -d (slurp (io/resource "serviceentry_8671.json")))

Reply:
Yes. There is a ticket to update that documentation seperate from this ticket